MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/GiniImage.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  GiniImage.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+
+/**
+ The GiniImage enumeration provides a convenient way to manage image assets within the Gini SDK, supporting customization for both light and dark modes. Each case in the enumeration represents a specific image asset used by the SDK.
+
+ - Note: The raw values for each case correspond to the image asset names in the asset catalog.
+ */
+
+public enum GiniImage: String {
```

Comment:
[Clarification]: GiniImage is quite general and might be used in other places or SDK or even in a shared package that we plan so far, maybe then `GiniMerchantIcon` or `GiniMerchantImage`?

Reply:
Thanks for review, done